app.js

```diff
@@ -101,12 +115,12 @@ app.get('/rideInfo/:rideId', function(req, res){
   mongoose.model('rideInfo').find({ 'rideId': req.params.rideId }, function (err, result) {
     if (err) {
       res.send(404, { 'success': false, error: { 'message': err } });
-      console.log('ERRORED at GET /   , ');
-      return console.error(err);
+      log.error('ERRORED at GET /   , ' + err);
```

Comment:
try to use `log.error(err);` looks like bunyan automatically parses it and adds to log.